wordpress-mobile/WordPress-iOS
# Update Me tab to use Gravatar SDK

When working on https://github.com/wordpress-mobile/WordPress-iOS/pull/23533, I noticed that me newly uploaded gravatar wasn't loading and wasn't being displayed in the tab bar. I reported the issue to the Gravatar team. In the meantime, I replaced the legacy code with the Gravatar SDK.

## To test:

- Upload a new profile picture
- Verify that it gets displayed in the tab bar

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
